src/sentry/mail/adapter.py

```diff
@@ -108,20 +107,16 @@ def get_sendable_user_objects(project):
         notifications for the provided project.
         """
         user_ids = project.member_set.values_list("user_id", flat=True)
-        users = user_service.get_many(filter=dict(user_ids=list(user_ids)))
+        actors = [RpcActor(id=uid, actor_type=ActorType.USER) for uid in user_ids]
 
-        # TODO: Do we need to use a notification service here?
-        controller = NotificationController(
-            recipients=users,
+        recipients = notifications_service.get_notification_recipients(
+            recipients=actors,
             project_ids=[project.id],
             organization_id=project.organization_id,
-            provider=ExternalProviderEnum.EMAIL,
-            type=NotificationSettingEnum.ISSUE_ALERTS,
-        )
-        return controller.get_notification_recipients(
             type=NotificationSettingEnum.ISSUE_ALERTS,
             actor_type=ActorType.USER,
-        )[ExternalProviders.EMAIL]
+        )
+        return recipients.get(ExternalProviders.EMAIL.name)
```

Comment:
I think I've done these substitutions correctly but would appreciate 👀 with more context.

Reply:
@markstory just use `get_notification_recipients_v2` s you don't have to

Reply:
Thanks!